File: starting_filesets/Java/Poker Hands/PokerHandsTest.java

import org.junit.*;
import static org.junit.Assert.*;

public class PokerHandsTest 
{
    @Test
    public void example()
    {
        int expected = 6 * 9;
        int actual = new PokerHands().answer();
        assertEquals(expected, actual);
    }
}

File: starting_filesets/C#/Poker Hands/PokerHandsTest.cs

using NUnit.Framework;

[TestFixture]
public class PokerHandsTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new PokerHands().Answer;
        Assert.AreEqual(expected, actual);
    }
}

